Use Sphinx :pep: syntax to link PEPs

kpinc opened the pull request:
Creating a news entry from the pull request as a separate commit.  Does this warrant an issue?

BrownTruck commented:
Hello!

I am an automated bot and I have noticed that this pull request is not currently able to be merged. If you are able to either merge the ``master`` branch into this pull request or rebase this pull request against ``master`` then it will eligible for code review and hopefully merging!

pradyunsg commented:
@kpinc Thanks for rebasing. This doesn't need a new issue.

pradyunsg commented on news/5491.doc:
```diff
@@ -0,0 +1 @@
+Use Sphinx :pep: syntax to link PEPs
```
Could you change this NEWS fragment to a .trivial? I don't think this warrants a NEWS entry -- change with no visible effect isn't worth calling out to end-users.

kpinc commented:
On Thu, 21 Jun 2018 04:45:19 -0700
Pradyun Gedam <notifications@github.com> wrote:

> Could you change this NEWS fragment to a .trivial?

Done.

Karl <kop@meme.com>
Free Software:  "You don't pay back, you pay forward."
                 -- Robert A. Heinlein

pradyunsg commented:
@kpinc Could you push some change to this branch? Possibly rebase it?

I think #5478 might be causing issues issues with this PR.

kpinc commented:
On Thu, 21 Jun 2018 10:55:24 -0700
Pradyun Gedam <notifications@github.com> wrote:

> @kpinc Could you push some change to this branch? Possibly rebase it?
> 
> I think #5478 might be causing issues issues with this PR.
> 

Rebased onto master.

Regards,

Karl <kop@meme.com>
Free Software:  "You don't pay back, you pay forward."
                 -- Robert A. Heinlein

kpinc commented:
On Thu, 21 Jun 2018 10:55:24 -0700
Pradyun Gedam <notifications@github.com> wrote:

> I think #5478 might be causing issues issues with this PR.

The only thing I could think of was that Travis is doing something
weird with bash and that it needs a "true" put in between
line 20 and 21 of tools/travis/run.sh, just to give it a
command that is "executed".  (As opposed to a built-in.)
(This branch being taken, perhaps, because it's "trivial"
and so not marked as a doc.  ??)

But this is entirely wishful
thinking on my part because I'm really ignorant
of Travis.  Not to mention what I don't know
about all the other moving parts.

Regards,

Karl <kop@meme.com>
Free Software:  "You don't pay back, you pay forward."
                 -- Robert A. Heinlein

pradyunsg commented:
@kpinc Sorry but could you rebase/merge again, I think I've removed the offending snippet of CI setup.

kpinc commented:
On Fri, 22 Jun 2018 09:50:14 -0700
Pradyun Gedam <notifications@github.com> wrote:

> @kpinc Sorry but could you rebase/merge again, I think I've removed
> the offending snippet of CI setup.

No worries.  Rebased onto master.

Regards,

Karl <kop@meme.com>
Free Software:  "You don't pay back, you pay forward."
                 -- Robert A. Heinlein

pradyunsg commented:
Cool. That worked out. Appveyor is looking at this PR again.
